LDAR_Sim/external_sensors/METEC_NO_WIND.py

```diff
@@ -0,0 +1,149 @@
+                venting (float): same as input
+                found_leak (boolean): Did the crew find at least one leak at the site
+
+
+    """
+
+    missed_leaks_str = '{}_missed_leaks'.format(self.config['label'])
+    equip_measured_rates = []
+    site_measured_rate = 0
+    found_leak = False
+    n_leaks = len(covered_leaks)
+    mdl = self.config['sensor']['MDL']
+
+    if self.config["measurement_scale"] == "site":
+        # factor of 3.6 converts g/s to kg/h
+        rate = covered_site_rate * 3.6
+        prob_detect = 1/(1+np.exp(mdl[0]-mdl[1]*rate))
+        if rate <= (mdl[2]*3.6):
+            site[missed_leaks_str] += n_leaks
+            self.timeseries[missed_leaks_str][self.state['t'].current_timestep] += n_leaks
+        elif np.random.binomial(1, prob_detect):
+            found_leak = True
+            site_measured_rate = measured_rate(
+                covered_site_rate, self.config['sensor']['QE'])
+        else:
+            site[missed_leaks_str] += n_leaks
+            self.timeseries[missed_leaks_str][self.state['t'].current_timestep] += n_leaks
+    elif self.config["measurement_scale"] == "equipment":
+        for rate in covered_equipment_rates:
+            m_rate = measured_rate(rate, self.config['sensor']['QE'])
+            rate = m_rate * 3.6
+            prob_detect = 1 / \
+                (1+np.exp(mdl[0]-mdl[1]*rate))
+            if rate <= (mdl[2]*3.6):
+                m_rate = 0
+            elif np.random.binomial(1, prob_detect):
+                found_leak = True
+            else:
+                m_rate = 0
+            equip_measured_rates.append(m_rate)
+            site_measured_rate += m_rate
+        if not found_leak:
+            site[missed_leaks_str] += n_leaks
+            self.timeseries[missed_leaks_str][self.state['t'].current_timestep] += n_leaks
+    elif self.config['measurement_scale'] == 'component':
+        for leak in covered_leaks:
+            rate = leak['rate'] * 3.6
+            prob_detect = 1 / \
+                (1+np.exp(mdl[0]-mdl[1]*rate))
+            if prob_detect >= 1:
+                prob_detect = 1
+            if rate <= (mdl[2]*3.6):
+                site[missed_leaks_str] += 1
+                self.timeseries[missed_leaks_str][self.state['t'].current_timestep] += 1
+            elif np.random.binomial(1, prob_detect):
+                found_leak = True
+                meas_rate = measured_rate(
+                    leak['rate'], self.config['sensor']['QE'])
+                is_new_leak = update_tag(
+                    leak,
+                    meas_rate,
+                    site,
+                    self.timeseries,
+                    self.state['t'],
+                    self.config['label'],
+                    self.id,
+                    self.parameters
+                )
+                if is_new_leak:
+                    site_measured_rate += meas_rate
```

Comment:
Could you explain why existing leaks are not included in the site measured rate? In my mind it makes sense to include since they are still leaking.

Reply:
This is a bad variable name, carried over from using one of the existing sensors as a template.

Reply:
This value is how the results are reported in the outputs file. Changing it to include all the existing leaks would result in a large change - therefore decided to keep as is.